tests/integration/docusaurus/tutorials/getting-started/getting_started.py

```diff
@@ -105,17 +105,51 @@
 ]
 # </snippet>
 
-profiler = UserConfigurableProfiler(
-    profile_dataset=validator,
-    excluded_expectations=None,
-    ignored_columns=ignored_columns,
-    not_null_only=False,
-    primary_or_compound_key=False,
-    semantic_types_dict=None,
-    table_expectations_only=False,
-    value_set_threshold="MANY",
-)
-suite = profiler.build_suite()
+data_assistant_result: DataAssistantResult = context.assistants.onboarding.run(
```

Comment:
Similar story here. Including all settings even if they are not used.. to help documentation.